Loop/Managers/LoopAppManager.swift

```diff
@@ -83,6 +83,8 @@ class LoopAppManager: NSObject {
         }
 
         registerBackgroundTasks()
```

Comment:
Registering for remote notifications on each startup -- this was discussed above and also in https://github.com/LoopKit/Loop/issues/1536.